config/database.yml

```diff
@@ -7,18 +7,22 @@
 default: &default
   adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  host: <%= ENV['POSTGRES_HOST'] %>
+  username: <%= ENV['POSTGRES_USER'] %>
+  password: <%= ENV['POSTGRES_PASSWORD'] %>
   timeout: 5000
 
 development:
   <<: *default
-  database: db/development.postgresql
+  database: <%= ENV['POSTGRES_DB'] %>
+
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test.postgresql
+  database: <%= ENV['POSTGRES_TEST_DB'] %>
```

Comment:
`[Optional]` for development and test environments it is better if you use explicit values instead of environment variables, we can check rapidly.